src/main/java/org/breedinginsight/api/v1/controller/ProgramController.java

```diff
@@ -88,6 +88,8 @@ public ProgramController(ProgramService programService, ProgramUserService progr
 
     @Get("/programs{?queryParams*}")
     @Produces(MediaType.APPLICATION_JSON)
+    @ProgramSecured(roles = {ProgramSecuredRole.SYSTEM_ADMIN, ProgramSecuredRole.READ_ONLY, ProgramSecuredRole.PROGRAM_ADMIN
```

Comment:
`ProgramSecuredAnnotationRuleIntegrationTest::getAllProgramsAdmin`,`ProgramSecuredAnnotationRuleIntegrationTest::getAllProgramsMember`, `ProgramControllerIntegrationTest::getProgramsSuccess`, `ProgramControllerIntegrationTest::getProgramQuery` failing with error message `Endpoint does not have program id to check roles against` with annotation added on this method.

Reply:
Fixed